### PR TITLE
X2-9688 added loading state for Arrival picker dates

### PR DESCRIPTION
**Pull request by SemenStruchev**
https://xola01.atlassian.net/browse/X2-9688

### CODE REVIEW

File: src/components/Dot/Dot.jsx [hunk 1]
```diff
@@ -15,6 +15,7 @@ const sizes = {
     small: "h-1 w-1",
     medium: "h-1.5 w-1.5",
     large: "h-2 w-2",
+    "extra-large": "h-8 w-8",
```
Comment (rushi) [inline src/components/Dot/Dot.jsx:18]: Make it `xlarge`

File: src/components/DatePicker/Day.jsx [hunk 2]
```diff
@@ -2,13 +2,31 @@ import clsx from "clsx";
...
+                <div className="mr-2 flex animate-pulse items-center justify-center">
+                    <Dot className="ui-day-content pointer-events-none !cursor-not-allowed" color="secondary" />
+                </div>
+            );
+        }
+
         return <DayContent selectedDate={selectedDate} date={date} getContent={getContent} />;
     }
 
+    if (isLoading) {
```
Comment (rushi) [inline src/components/DatePicker/Day.jsx:22]: What is the difference between this and the one above

**Review by rushi — CHANGES_REQUESTED** [review]
Can you also put screenshots of the change

File: src/components/DatePicker/Day.jsx (hunk 2 above)
Comment (SemenStruchev) [reply src/components/DatePicker/Day.jsx:22]: Firtst one displayed for Custom content and Dots there have bigger size, Second one displayed for Dates without custom content, only with the number of the month and they have fixed size

**Comment by SemenStruchev** [comment]
Before:
image
After:
image

**Pull request merged**